src/js/pages/Index.js

```diff
@@ -72,6 +73,7 @@ var Index = React.createClass({
 
     ConfigStore.fetchCCID();
     ConfigStore.addChangeListener(EventTypes.CONFIG_ERROR, this.onConfigError);
+    compare.subscribe(newVersion => console.log(newVersion));
```

Comment:
Maybe we could add a TODO comment here to clean up the log once this is handled. Also, we need to assign this line to a `Subscription` object so that we can `unsubscribe` somewhere (could be in `componentWillUnmount`)

Reply:
Good points, the `console.log` was just for testing and unsubscribing should be possible.